src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inputStatement;
+        int endFlag = 0;
+        String separatingLine = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+        Task[] listOfItems = new Task[100];
```

Comment:
Try to avoid the use of a magic number here. Using a named constant makes the code easier to understand because the name tells us more about the meaning of the number.